tests/unittests/PartitionerTest.cpp

```diff
@@ -790,3 +790,22 @@ TEST_F(PartitionerTest, graphMemInfoCalculation2) {
   res2 = getGraphMemInfo(nodes2);
   EXPECT_EQ(res2, GraphMemInfo(96, 32, 544));
 }
+
+/// This one test the memoryUsageValidation in Partitioner : the memory usage of
+/// one single node is larger than the given device memory.
+TEST_F(PartitionerTest, memoryUsageValidation1) {
+  auto *input1 =
+      mod_.createPlaceholder(ElemKind::FloatTy, {2, 10}, "input1", false);
+  auto *input2 =
+      mod_.createPlaceholder(ElemKind::FloatTy, {10, 16}, "input2", false);
+  auto *mul0 = F_->createMatMul("mul0", input1, input2);
+  auto *save = F_->createSave("ret", mul0);
```

Comment:
do we need save node if there is no compilation here?

Reply:
No, the "save" node is not necessary. Just want to contain more than one node in a function :)

Reply:
it's fine, maybe just do F_->createSave("ret", mul) and remove (void)save.